config/initializers/locale.rb

```diff
@@ -6,4 +6,5 @@
   config.i18n.load_path += Dir[Rails.root.join('app', 'javascript', 'skins', '*', '*', 'names.{rb,yml}').to_s]
   config.i18n.load_path += Dir[Rails.root.join('app', 'javascript', 'skins', '*', '*', 'names', '*.{rb,yml}').to_s]
   config.i18n.load_path += Dir[Rails.root.join('config', 'locales-glitch', '*.{rb,yml}').to_s]
+  config.i18n.load_path += Dir[Rails.root.join('config', 'locales-polyam', '*.{rb,yml}').to_s]
```

Comment:
I'm unsure whether I should add locales-polyam or replace locales-glitch.
Both works, but I'm not exactly sure what this is for.

Reply:
Oops, definitely keep as is as otherwise glitch locales are not loaded.
Should've checked more thoroughly.